common-flow.md

```diff
@@ -56,8 +60,16 @@ interpreted as described in [RFC 2119](https://tools.ietf.org/html/rfc2119).
        release/production" state to reduce any friction with creating a new
        release.
 3. Change Branches
-    1. Each change (feature, bugfix, etc.) MUST be performed on separate
-       branches that SHOULD be referred to as "change branches".
+    1a. Before a project is Live;
+       - each change (feature, bugfix, etc.) SHOULD be performed on separate 
+       branches that SHOULD be referred to as "change branches". You MAY
+       perform small (typically single-changeset) changes directly to the 
+       Master Branch, subject to not breaking other requirements.
```

Comment:
What does 

> not breaking other requirements

 mean?

Reply:
@michaelchart Is the local scope of 'of this spec' not implied? ie don't violate any of the other MUSTs

Reply:
Do we need to say "subject to not breaking other requirements" - just seems slightly unnecessary?